app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -30,7 +30,7 @@ class MySpacesSection extends ConsumerWidget {
             style: Theme.of(context).textTheme.titleMedium,
           ),
         ),
-        spaces.isEmpty
+        !spaces.isEmpty
```

Comment:
This seems wrong

Reply:
It was change when I was testing empty data state locally and was committed by mistake by already fixed on last committ.
https://github.com/acterglobal/a3/pull/1569/commits/765118d855638fb96f461a3844d0e9f12bacae67